res/database/Default/user/scr0llbaer_screenplays_adaptations.xml

```diff
@@ -1758,14 +1784,9 @@
 						<en>March of the Wooden Soldiers</en>
 					</title_original>
 					<description>
-						<de>Der %ROLE4% kandidiert weiter für das Amt des Bürgermeisters von Fabletown, während die Gemeinde sich mit der scheinbaren Flucht von %ROLENAME5% aus der Heimat beschäftigt. Der Feind sendet seine ersten Truppen nach Fabletown, um einen Angriff zu beginnen.</de>
-						<en>%ROLE4% runs for Mayor of Fabletown while the community deals with the apparent escape from the Homelands of %ROLENAME5%. The Adversary sends his first troops into Fabletown to begin an assault.</en>
+						<de>Der %ROLE4% kandidiert weiter für das Amt des Bürgermeisters von Fabletown, während die Gemeinde sich mit der scheinbaren Flucht von Rotkappe aus der Heimat beschäftigt. Der Feind sendet seine ersten Truppen nach Fabletown, um einen Angriff zu beginnen.</de>
```

Comment:
Why doesn't the old reference work, anymore?

Reply:
Added more seasons, now larger cast list, can only reference up to 7.

Reply:
Just wondering, is it reasonable to have such a large cast?
I would not extend the current code to make more roles referrable. With the intended new function syntax `${}`, there should be no restictions out of the box.

Reply:
It's what the source material is, rather large ensemble, even had to leave out many (additional well-known fairy-tale) characters for now and skipped any supporting roles. And almost all characters are mentioned eventually, but also appear in most / earlier episodes too, so for my own sanity I just applied the (roughly) same cast list for each season.